src/openforms/formio/management/commands/formio_makemessages.py

```diff
@@ -0,0 +1,151 @@
+        report_unknown = options.get("report_unknown")
+        report_output = options.get("report_output")
+
+        translations_filepath = options.get("file") or os.path.join(
+            settings.DJANGO_PROJECT_DIR, "js/lang/formio/nl.json"
+        )
+
+        if not os.path.exists(translations_filepath):
+            raise CommandError(
+                f"Translations file {translations_filepath} does not exist."
+            )
+
+        # Load existing translations.
+        try:
+            with open(translations_filepath, "r", encoding="utf-8") as f:
+                existing_translations = json.load(f)
+        except Exception as exc:
+            self.stderr.write(f"Could not load translations: {exc}")
+
+        if self.verbosity > 0:
+            self.stdout.write(f"Using translations file: {translations_filepath}")
+
+        # Find new translations.
+        formio_folders = options.get("formio") or [
+            os.path.join(settings.DJANGO_PROJECT_DIR, "js/components"),
+            os.path.join(settings.DJANGO_PROJECT_DIR, "static/bundles"),
+        ]
+        for formio_folder in formio_folders:
+            if not os.path.exists(formio_folder):
+                raise CommandError(f"FormIO folder {formio_folder} does not exist.")
+            if self.verbosity > 0:
+                self.stdout.write(f"Using FormIO folder: {formio_folder}")
+
+        # This entry helps to identify newly found translations.
+        found_translations = {
+            "---------- NEW ----------": "---------- NEW ----------",
+        }
+
+        for formio_folder in formio_folders:
+            for root, subFolders, files in os.walk(formio_folder, topdown=True):
+                for file in files:
+                    if file.endswith(".js"):
+                        filepath = os.path.join(root, file)
+                        if self.verbosity > 1:
+                            self.stdout.write(f"Scanning file: {filepath}")
+
+                        result = self._find_translations(filepath, no_empty_values)
+                        found_translations.update(result)
+
+        # Report unknown
+        diff = set(existing_translations).difference(set(found_translations))
+        self.stdout.write(
+            f"WARNING: Could not find matches for {len(diff)} existing translations."
+        )
+        if report_unknown:
+            for item in diff:
+                self.stdout.write(f"Not found: {item}")
+
+        # Merge
+        result = {}
+
+        # Use order of existing translations.
+        result.update(existing_translations)
+
+        # Update with new and existing translations (new will go at the bottom)
+        result.update(found_translations)
+
+        # Overwrite the existing translation keys with their existing
+        # translation values
+        result.update(existing_translations)
+
+        if report_output:
+            self.stdout.write(json.dumps(result, indent=2))
```

Comment:
I think you can use `json.dump(result, self.stdout, indent=2)` too here, but I'm not sure

Reply:
Just tested it. It works, but formatting is barfed. Almost every word is put on a new line.